website/versioned_docs/version-v1.x/installation.md

```diff
@@ -2,7 +2,7 @@
 title: Installing MACI
 description: How to install MACI
 sidebar_label: Installation
-sidebar_position: 3
+sidebar_position: 4
```

Comment:
maybe not only change `sidebar_position` in `installation.md` and `overview.md`, but also others?
Now it looks ok might because `primitives.md` originally appeared after `installation.md` (both of them having `sidebar_position: 4`